sdcm/sct_provision/aws/instance_parameters_builder.py

```diff
@@ -82,7 +83,9 @@ def InstanceType(self) -> str:  # pylint: disable=invalid-name
 
     @property
     def Placement(self) -> Optional[AWSPlacementInfo]:  # pylint: disable=invalid-name
-        return AWSPlacementInfo(AvailabilityZone=self._region_name + self._availability_zones[self.availability_zone])
+        return AWSPlacementInfo(
```

Comment:
what about the case we don't use placement groups ?

Reply:
it will be 'None' as previous: 
```
class AWSPlacementInfo(BaseModel):
    AvailabilityZone: str
    GroupName: str = None
    Tenancy: Literal['default', 'dedicated', 'host'] = 'default'

```


this case is used  in MonitoringCluster:
```
# disable placement group for monitor nodes, because it doesn't need low-latency network performance
_USE_PLACEMENT_GROUP = False
```

Reply:
o.k. was confusing a bit

why don't we do the placement for monitor ? would it hurt us to do so ? or just cause it's irrelevant ?